blocks/components/editable/index.js

```diff
@@ -101,14 +110,26 @@ export default class Editable extends wp.element.Component {
 		this.props.onFocus();
 	}
 
-	onChange() {
-		if ( ! this.editor.isDirty() ) {
+	isActive() {
+		return tinymce.EditorManager.activeEditor.id === this.editor.id;
```

Comment:
I think toward a goal of keeping this component self-contained (not relying on globals), it might be reasonable to recreate the behavior that sets the active editor; i.e. bind to the editor's `activate`, `deactivate` events:

https://github.com/tinymce/tinymce/blob/13d40cd/src/core/src/main/js/EditorManager.js#L722-L740